lib/voyager/clients/twitter_client.rb

```diff
@@ -183,6 +183,18 @@ def recent_tweets(options={})
       get("/statuses/user_timeline.json", options)
     end
 
+    # ============================================================================
+    # Metrics Methods - These act on Tweets
+    # ============================================================================
+
+    def follower_count(options = {})
+      get("/users/#{options[:profile_id]}", options[:params])
+    end
+
+    def bulk_metrics(params)

```

Comment:
you may need a default for params and I would rename it to options since that is that pattern we use in the other methods.  

`def bulk_metrics(options={})`